core/include/gnuradio-4.0/YamlPmt.hpp

```diff
@@ -8,6 +8,7 @@
 #include <cstdint>
 #include <cstdio>
 #include <expected>
+#include <iomanip>
```

Comment:
correct solution and already included in: https://github.com/fair-acc/gnuradio4/pull/483